tests/graph/test_basic.py

```diff
@@ -522,6 +523,19 @@ def test_graph_inputs():
     assert res_list == [r3, r1, r2]
 
 
+def test_explicit_graph_inputs():
+    x = pt.fscalar()
+    y = pt.constant(2)
+    z = shared(1)
+    a = pt.sum(x + y + z)
+    b = pt.true_div(x, y)
+
+    res = list(explicit_graph_inputs([a]))
+    res1 = list(explicit_graph_inputs(b))
+
+    assert res, res1 == [x]
```

Comment:
This assert looks wrong

Reply:
Here I am checking that constant and shared variables are excluded from the output given by ```explicit_graph_inputs```. What else would you suggest to do here?

Reply:
I don't think this assert is doing what you think. The assert syntax is `assert, execute y` usually `assert x, print(y)`.

Reply:
Ohh I understand